docs/developers/developer-reference/sc-contract-calls.md

```diff
@@ -183,59 +500,56 @@ fn caller_endpoint(&self) {
 ```
 
 Notice how the callback now has an argument:
+
 ```rust
 self.callbacks().my_endpoint_callback(caller)
 ```
 
 You can then use `original_caller` in the callback like any other function argument.
 
+
+
 [comment]: # (mx-context-auto)
 
-### Payments in proxy arguments
+### Promises
 
-Let's say you want to call a `#[payable]` endpoint, with this definition:
-```rust
-#[payable("*")]
-#[endpoint(myEndpoint)]
-fn my_payable_endpoint(&self, arg: BigUint) -> BigUint {
-	let payment = self.call_value().egld_or_single_esdt();
-	// implementation
-}
+Promises (or multi-async calls) are a new feature that will be introduced in mainnet release 1.6. They are very similar to the old asynchronous calls, with the difference that launching them does not terminate current execution, and there can be several launched from the same transaction.
+
+:::caution
+Because this feature is currently not available on mainnet, contracts need to enable the "promises" feature flag in `Cargo.toml` to use the functionality:
```

Comment:
Don't know what would be the best solution here to avoid the need to modify the docs after the release of 1.6

Reply:
We will modify them.